DigitalLearningSolutions.Data/Models/User/ChooseACentreAccountViewModel.cs

```diff
@@ -7,10 +7,11 @@ public class ChooseACentreAccountViewModel
         public readonly int CentreId;
         public readonly string CentreName;
         public readonly bool IsActiveAdmin;
-        public readonly bool IsDelegate;
-        private readonly bool isCentreInactive;
         private readonly bool isApprovedDelegate;
+        private readonly bool isCentreInactive;
+        public readonly bool IsDelegate;
         private readonly bool isInactiveDelegate;
+        private readonly bool isUnverifiedEmail;
 
         public ChooseACentreAccountViewModel(

```

Comment:
Would you mind making another folder in the Data project for ViewModels and sticking this in there. It would be good to reflect the file structure of Web/ViewModels too.

Reply:
Sure, done!